rootdir/ueventd.shinano.rc

```diff
@@ -155,3 +155,7 @@
 /sys/devices/system/cpu/cpuquiet/rqbalance/balance_level             0660 system system
 /sys/devices/system/cpu/cpuquiet/rqbalance/nr_run_thresholds         0660 system system
 /sys/devices/system/cpu/cpuquiet/rqbalance/nr_down_run_thresholds    0660 system system
+
+# BT/FM/ANT+ V4L2
+/dev/brcm_bt_drv          0660   bluetooth  net_bt_stack
+/dev/radio0               0666   root       root
```

Comment:
@humberos should not be https://github.com/sonyxperiadev/device-sony-rhine/blob/master/rootdir/ueventd.rhine.rc#L14 ??? Due to https://github.com/sonyxperiadev/packages-apps-FMRadio/commit/9025dac0e7af82ea8b6ffb0e47afe90c52493106 .....